src/login/login-menu.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+
+import { useContext } from 'login/context'
+import { LoginButton } from 'login/login-button'
+
+export const LoginMenu = () => {
+  const { user } = useContext()
+
+  return <div>
+    <LoginButton user={user} />
+  </div>
+}
```

Comment:
Would the name `Menu` be more relevant for this component?

Reply:
Should we also make sure we rename the file also?

Reply:
Done